pyjobshop/solvers/ortools/Variables.py

```diff
@@ -474,6 +519,55 @@ def _make_sequence_variables(self) -> dict[ResourceIdx, SequenceVar]:
 
         return variables
 
+    def _make_overlap_variables(self) -> list[list[BreakOverlapVar]]:
+        """
+        Creates the overlap variables.
+        """
+        model, data = self._model, self._data
+        variables: list[list[BreakOverlapVar]] = []
+
+        for mode_idx, mode in enumerate(data.modes):
+            # For single-resource modes, breaks map directly to individual
+            # resource breaks. For multi-resource modes, breaks may represent
+            # combined breaks (e.g., when resources have overlapping breaks).
+            all_breaks: list[tuple[int, int]] = []
+            for res_idx in mode.resources:
+                all_breaks.extend(data.resources[res_idx].breaks)
+            breaks = utils.merge(all_breaks)
+
+            mode_var = self._mode_vars[mode_idx]
+            overlap_vars = []
+
+            for start, end in breaks:
+                before = model.new_bool_var("")
+                after = model.new_bool_var("")
+                has_overlap = model.new_bool_var("")
+
+                # If mode is selected, then (after OR before) XOR has_overlap.
+                model.add_bool_or(after, before, has_overlap).only_enforce_if(
+                    mode_var
+                )
+                model.add_implication(after, ~has_overlap)
+                model.add_implication(before, ~has_overlap)
+
```

Comment:
Why is it not possible to make sure that all three bools are inactive when mode var is inactive?